src/workerd/util/sqlite.c++

```diff
@@ -860,6 +860,8 @@ void SqliteDatabase::Query::nextRow() {
 
   int err = sqlite3_step(statement);
   if (err == SQLITE_DONE) {
+    db.addRowsRead(sqlite3_stmt_status(statement, LIBSQL_STMTSTATUS_ROWS_READ, 0));
+    db.addRowsWritten(sqlite3_stmt_status(statement, LIBSQL_STMTSTATUS_ROWS_WRITTEN, 0));
```

Comment:
Do these statement-level counters include operations from any triggers that the statement, er, triggered? Worth testing?

Reply:
There's a test, but I had the tests split into a separate commit from the implementation for some reason. I've merged them.

(And yes, any triggers' IO is accounted for.)